.taskcluster.yml

```diff
@@ -62,6 +62,38 @@ tasks:
       owner: "{{ event.head.user.email }}"
       source: "{{ event.head.repo.url }}"
 
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+    workerType: "{{ taskcluster.docker.workerType }}"
+    scopes:
+      - secrets:get:repo:github.com/mozilla/balrog:coveralls
+    payload:
+      maxRunTime: 1200
+      image: "mozillareleases/python-test-runner"
+      env:
+        NO_VOLUME_MOUNT: 1
+      command:
+        - "/bin/bash"
+        - "-c"
+        - "git clone $GITHUB_HEAD_REPO_URL balrog && cd balrog && git checkout $GITHUB_HEAD_BRANCH && bash run-tests.sh backend py36"
```

Comment:
Now that it's out, can we move up to 3.7? May as well get as up-to-date as possible.